### DISCUSSION TITLE
Consertando o Para-brisa

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#401] [Imagens]
Post de referência: https://www.vish.blog.br/imagens/consertando-o-para-brisa/

**Comment by maryon-telvanni** [comment]
Hmmm... Gohan e Tenshinhan?

**Comment by brunoaugusto** [comment]
> [Maryon Telvanni](</perfil/?user=maryon-telvanni>) disse [sábado, 30 de abril de 2022](<https://www.vish.blog.br/imagens/consertando-o-para-brisa/#comment_1113993129>):

> Hmmm... Gohan e Tenshinhan?

Não manjo de Dragon Ball, mas pelo cabelo de um e as calças verdes do outro, é bem possível
